src/messages.cpp

```diff
@@ -142,6 +142,40 @@ namespace quicr::messages {
         return true;
     }
 
+    Bytes& operator<<(Bytes& buffer, const TrackNamespace& ns)
+    {
+        const auto& entries = ns.GetEntries();
+
+        Bytes payload;
+        for (const auto& entry : entries) {
+            payload << UintVar(entry.size());
+            payload << entry;
+        }
+
+        buffer << UintVar(payload.size());
```

Comment:
this should be the following on the wire
Varint(num_entries) entry1, entry2, ... 

``` buffer << UintVar(entries.size()) ````

Reply:
I had started with that, but then didn't need it. I can add it back.

Reply:
i can see that . The spec says this 

x(tuple):
Indicates that x is a tuple, consisting of a variable length integer encoded , followed by that many variable length tuple fields, each of which are encoded as variable length integer encoding, followed by that many bytes of binary data